evals/elsuite/audio/eval.py

```diff
@@ -70,6 +73,7 @@ def load_dataset(self):
         ds = load_hf_dataset(self.dataset, evals.eval._MAX_SAMPLES).cast_column(
             "audio", Audio(sampling_rate=DEFAULT_SAMPLE_RATE)
         )
+        ds = ds.filter(IsAudioLengthInRange(self.max_audio_duration))
```

Comment:
The audio isn't always in an "audio" column so this approach won't work for say, the Tools task (and in Tools there are multiple audio utterances per sample). Maybe allow the column name and/or a lambda to be passed in to allow for this?

Reply:
Hmm, I was wondering how that was being handled elsewhere. I'll take a look.

Reply:
I added a `_keep_sample` function to handle this separately.